docs/compression_algorithms/BatchnormAdaptation.md

```diff
@@ -0,0 +1,9 @@
+#### Batch-norm statistics adaptation
+
+After the compression-related changes in the model have been committed, the statistics of the batchnorm layers (per-channel rolling means and variances of activation tensors) can be updated by passing several batches of data through the model before the fine-tuning starts. 
+This allows to correct the compression-induced bias in the model and reduce the corresponding accuracy drop even before model training. 
+This option is common for quantization, magnitude sparsity and filter pruning algorithms. 
+It can be enabled by setting a non-zero value of `num_bn_adaptation_samples` in the `batchnorm_adaptation` section of the `initializer` configuration - see [NNCF config schema](https://openvinotoolkit.github.io/nncf/) for reference.
```

Comment:
a piece of the config related to this algorithm would be helpful here.

Reply:
Done